data/src/entity_manager/migrate_inheritance.py

```
@@ -0,0 +1,515 @@
+    )
+
+    args = parser.parse_args()
+
+    # Create migrator from config
+    migrator = create_migrator_from_config(args.config)
+
+    if args.dry_run:
+        print("DRY RUN MODE - No data will be migrated")
+        print(f"Source: {migrator.source_connection_string}")
+        print(f"Target: {migrator.target_connection_string}")
+        print(f"Entity classes to migrate:")
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)